api/model/form/location.go

```diff
@@ -310,6 +310,7 @@ var (
 	}
 
 	countries = []string{
+		"POSTOFFICE",
```

Comment:
this seems strange. I'm sure it's not a mistake but just curious, why is this here?

Reply:
That is the "country" value when the address is a APO/FPO/DPO. 

We can probably look at removing it here and make sure it is not included in the validation.